infrastructure/docker/entrypoint.sh

```
@@ -0,0 +1,30 @@
+fi
+
+check_command() {
+    if ! command -v "$1" >/dev/null 2>&1; then
+        echo "Error: $1 command not found"
+        exit 1
+    fi
+}
+
+check_command gosu
+check_command python3
+fi
```

Shellcheck

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)